tidb-cloud/changefeed-overview.md

```diff
@@ -4,13 +4,56 @@ title: Changefeed
 
 # Changefeed
 
-TiDB Cloud provides the following changefeeds to help you stream data from TiDB Cloud:
+> **Note:**
+>
+> To use the Changefeed feature, make sure that your TiDB cluster version is v6.4.0 or later and the TiKV node size is at least 8 vCPU and 16 GiB.
+>
+> Currently, tidb cloud only allows up to 10 changefeeds per cluster, regardless of kafka and mysql sink types.
+>
+> For [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta), the changefeed feature is unavailable.
+
```

Comment:
Can you add a few sentences to introduce changefeed? For example, what is changefeed, how can it help your business...

Reply:
Yes